tests/test_utils/system_tests_class.py

```diff
@@ -106,6 +107,11 @@ def _restore_dags_from_temporary_directory(
 
 
 class SystemTest(TestCase, LoggingMixin):
+    executor = LoggingCommandExecutor()
```

Comment:
LoggingCommandExecutor contains only static methods. It may be worth changing to raw functions so that you do not have to create an instance of this class.

Reply:
I would prefer to do this as next PR